File: WIP/1588-Sum_of_all_Odd_Length_Subarrays.py
class Solution:
    def sumOddLengthSubarrays(self, arr: List[int]) -> int:
        sum = 0

        for i in range(1, len(arr) + 1, 2):
            for j in range(0, len(arr)):
                if j + i >= len(arr):
                    break

                for k in range(0, i):
                    sum += arr[j + k]

        return sum

File: random_solutions/Easy/1588-Sum_of_All_Odd_Length_Subarrays.ts
function sumOddLengthSubarrays(arr: number[]): number {
  // Time { O(n**3), θ(nlogn), Ω(n) } and Space { O(n**3), θ(nlogn), Ω(n) }
  let answer = 0;

  for (let i = 1; i <= arr.length; i += 2) {
    for (let j = 0; j < arr.length; j++) {
      let currArr: number[] = [];

      for (let k = j; k < j + i; k++) {
        if (k == arr.length) break;
        currArr.push(arr[k]);
      }

      if (currArr.length != i) break;

      for (let num of currArr) answer += num;
    }
  }

  return answer;
}
